compatibility/deps.bzl

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# The dependencies of the daml workspace.
+# This allows using the daml workspace externally
+# from another bazel workspace.
+#
+# For example, another Bazel project can depend on
+# targets in the daml repository by doing:
+# ---
+# local_repository(
+#   name = "com_github_digital_asset_daml",
+#   path = "/path/to/daml"
+# )
+# load("@com_github_digital_asset_daml//:deps.bzl", "daml_deps")
+# daml_deps()
+# ---
+#
+# A 3rd-party consumer would also need to register relevant
+# toolchains and repositories in order to build targets.
+# That is, copy some setup calls from WORKSPACE into the
+# other WORKSPACE.
+#
+# Make sure to reference repository local files with the full
+# prefix: @com_github_digital_asset_daml//..., as these won't
+# be resolvable from external workspaces otherwise.
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+rules_haskell_version = "cf6beb4e3748dce6dc401a95ab8cacb803c5d944"
+rules_haskell_sha256 = "cadb4ceb63994d86343fb407a9b5eaab363af428850d1698d6c6132690ccfb71"
+rules_nixpkgs_version = "c966bb8bd335f1e244c03efe6e7a1afa9784038e"
+rules_nixpkgs_sha256 = "ccafea4fc4d5fa2ddba2882f76728558bfe2c12657f7f56078ece43a31761148"
```

Comment:
Maybe import those from `@daml//:deps.bzl`? To make it easier to keep them in sync.

Reply:
Good idea, changed!